wp-content/themes/amplify/functions.php

```diff
@@ -45,3 +45,91 @@ function largo_parent_theme_enqueue_styles() {
 	);
 }
 add_action( 'wp_enqueue_scripts', 'largo_parent_theme_enqueue_styles', 20 );
+
+/**
+ * Add query vars specific to the Amplify child theme
+ * 
+ * @param array $vars The array of available query vars
+ * 
+ * @return array $vars The modified array of available query vars
+ */
+function amplify_add_query_vars( $vars ) {
+	
+	$vars[] = 'amplify-feed';
+	
+	return $vars;
+
+}
+add_filter( 'query_vars', 'amplify_add_query_vars' );
+
+/**
+ * Load the Amplify embed feed template if specific query param is present
+ * 
+ * @param str $template The path of the current template
+ * 
+ * @return str $template The path of the template to include
+ */
+function amplify_load_feed_template( $template ) { 
+
+    if ( ! is_admin() ) {
+        // if the params are set, use our amplify feed template if it exists
+		if( isset( $_GET['amplify-feed'] ) && locate_template( 'amplify-feed.php' ) ){
+			return get_stylesheet_directory().'/amplify-feed.php';
+		// else, continue with whatever template was being loaded
+		} else {
+			return $template;
+		}
+    }
+	return $template;
+	
+}
+add_filter( 'template_include', 'amplify_load_feed_template' );
+
+/**
+ * Add the "More link" input field to the tag edit page for 'lr-tags' taxonomy
+ * 
+ * @param obj $context The wp term that is being modified
+ */
+function amplify_add_saved_link_roundups_more_link_metabox( $context = '' ) {
+
+	if( 'lr-tags' == $context->taxonomy ){
+		
+		$term_more_link = get_term_link( $context, $context->taxonomy );
+
+		// Post ID here is the id of the post that Largo uses to keep track of the term's metadata. See largo_get_term_meta_post.
+		$post_id = largo_get_term_meta_post( $context->taxonomy, $context->term_id );
+
+		if( metadata_exists( 'post', $post_id, 'lr_more_link' ) ) {
+			$term_more_link = get_post_meta( $post_id, 'lr_more_link', true );
+		}
+
+		?>
+		<tr class="form-field">
+			<th scope="row" valign="top"><?php _e('More link', 'amplify'); ?></th>
+			<td>
+				<input class="widefat" id="more_link" name="lr_more_link" type="url" value="<?php echo $term_more_link; ?>" />
+				<p class="description">If the "More" link that is displayed needs to redirect to somewhere other than the tag archive page, change it here.</p>
```

Comment:
Make sure your strings are translatable.

Reply:
@benlk 👍 updated in https://github.com/INN/umbrella-inndev/pull/130/commits/37305df5350a65458d8eaa16730acc774dca60f8.